### PR TITLE
Progress Indicator

### PR DESCRIPTION
**Pull request by anhduc130**

### CODE REVIEW

File: src/components/ProgressIndicator.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const ProgressIndicator = () => (
+	<div className= "progressIndicator"></div>
+);
+
```
Comment (jordanschalm) [inline src/components/ProgressIndicator.js:26]: When you have a stateless component, you need to give it a display name after the component. Add `ProgressIndicator.displayName = 'ProgressIndicator'` here and your tests will pass ;)

File: src/components/ProgressIndicator.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const ProgressIndicator = () => (
+	<div className= "progressIndicator"></div>
```
Comment (jordanschalm) [inline src/components/ProgressIndicator.js]: spacing should be `... className="progressIndicator" ...`

File: bin/progressIndicatorStyling.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  width: 120px;
+  height: 120px;
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+  
+}
+
+@-webkit-keyframes spin {
+  0% { -webkit-transform: rotate(0deg); }
+  100% { -webkit-transform: rotate(360deg);   m }
```
Comment (jordanschalm) [inline bin/progressIndicatorStyling.css]: What's this `m` doing?

**Review by jordanschalm — CHANGES_REQUESTED** [review]
Couple minor changes. Good work!

File: bin/progressIndicatorStyling.css (hunk 3 above)
Comment (anhduc130) [reply bin/progressIndicatorStyling.css]: Oh! That's a bug :(...

File: src/components/ProgressIndicator.js (hunk 1 above)
Comment (anhduc130) [reply src/components/ProgressIndicator.js:26]: Thanks Jordan! Added it 💃

**Review by jordanschalm — APPROVED** [review]

**Pull request merged**